### PR TITLE
Extra Docker Automation

### PR DESCRIPTION
**Pull request by alexey-temnikov**

### CODE REVIEW

File: docker/sandbox-clientenv.sh [hunk 1]
```diff
@@ -8,6 +8,10 @@ function configure_db() {
 }
 
 function configure_hdfs() {
+  docker exec hdfs service ssh start
+  docker exec hdfs start-dfs.sh
+  sleep 10
```
Comment (jonathanl-bq) [inline docker/sandbox-clientenv.sh:13]: I guess the sleep is necessary, but I'm still curious to know why it's here.

File: docker/sandbox-clientenv.sh [hunk 2]
```diff
@@ -8,6 +8,10 @@ function configure_db() {
...
+#  docker exec hdfs hdfs dfsadmin -safemode forceExit
```
Comment (jonathanl-bq) [inline docker/sandbox-clientenv.sh:14]: Do we need this?

Comment (alexey-temnikov) [reply docker/sandbox-clientenv.sh:14]: No, needs to be removed.

File: docker/sandbox-clientenv.sh (hunk 1 above)
Comment (alexey-temnikov) [reply docker/sandbox-clientenv.sh:13]: Artifact from testing, suggesting to remove and confirm no impact.

**Pull request merged**